vm.go

```diff
@@ -731,3 +749,71 @@ func getSchemaVersion(opts *VirtualMachineOptions) hcsschema.Version {
 		Minor: 1,
 	}
 }
+
+func (vm *VirtualMachineSpec) GetDiskInformation(diskPath string) (controlNumber int, err error) {
```

Comment:
>func [](http://example.com/codeflow?start=0&length=4)

move these functions to line#722, keep them in group with other functions for VirtualMachineSpec.